.gitignore

```diff
@@ -240,3 +240,6 @@ samples/framework/docker/MVCRandomAnswerGenerator/containerImage
 .DS_Store
 _dependentPackages/
 !/xml/System.Net.Cache/
+
+# Ignore ionide caches
+.ionide/
```

Comment:
This has been added intentionally, tells git to ignore the cache folder created by the VS Code F# extension.